.github/workflows/ci-pr.yml

```diff
@@ -8,45 +8,34 @@ concurrency:
 jobs:
   ci-pr:
     name: CI-PR
-    needs: [test-minimal, test-full, msrv]
+    needs: [test]
     runs-on: ubuntu-latest
     steps:
       - name: Done
         run: exit 0
-  test-minimal:
-    name: Tests (Minimal)
-    env:
-      FLAGS: --no-default-features --features 'std cargo'
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu-latest, windows-latest]
-        rust: [stable]
-    runs-on: ${{ matrix.os }}
+  args:
+    name: Args
+    runs-on: ubuntu-latest
+    outputs:
+      minimal: ${{ steps.minimal.outputs.list }}
+      full: ${{ steps.full.outputs.list }}
     steps:
-      - name: Install rust
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ matrix.rust }}
-          override: true
-      - name: Cache Builds
-        uses: Swatinem/rust-cache@v1
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Compile
-        run: cargo test --no-run ${{ env.FLAGS }}
-      - name: Test
-        run: cargo test ${{ env.FLAGS }}
-  test-full:
-    name: Tests (Full)
-    env:
-      FLAGS: --features 'wrap_help yaml regex unstable-replace unstable-multicall unstable-grouped'
+      - id: minimal
+        name: Get args (Minimal)
+        run: echo "::set-output name=list::$(make features-minimal)"
+      - id: full
+        name: Get args (Full)
+        run: echo "::set-output name=list::$(make features-full)"
+  test:
+    name: Tests
+    needs: [args]
     strategy:
       fail-fast: false
       matrix:
+        rust: [1.54.0, stable]
```

Comment:
In #2802, we had agreed to only check the MSRV and not to do a full test run on it.

Reply:
I wanted to check something but forgot to move it back.